collector/netframework_clrexceptions.go

```diff
@@ -0,0 +1,132 @@
+// returns data points from Win32_PerfRawData_NETFramework_NETCLRExceptions
+// <add link to documentation here> - Win32_PerfRawData_NETFramework_NETCLRExceptions class
+package collector
+
+import (
+	"log"
+
+	"github.com/StackExchange/wmi"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+func init() {
+	Factories["netframework_clrexceptions"] = NewWin32_PerfRawData_NETFramework_NETCLRExceptionsCollector
+}
+
+// A Win32_PerfRawData_NETFramework_NETCLRExceptionsCollector is a Prometheus collector for WMI Win32_PerfRawData_NETFramework_NETCLRExceptions metrics
+type Win32_PerfRawData_NETFramework_NETCLRExceptionsCollector struct {
+	NumberofExcepsThrown       *prometheus.Desc
+	NumberofExcepsThrownPersec *prometheus.Desc
+	NumberofFiltersPersec      *prometheus.Desc
+	NumberofFinallysPersec     *prometheus.Desc
+	ThrowToCatchDepthPersec    *prometheus.Desc
+}
+
+// NewWin32_PerfRawData_NETFramework_NETCLRExceptionsCollector ...
+func NewWin32_PerfRawData_NETFramework_NETCLRExceptionsCollector() (Collector, error) {
+	const subsystem = "win32_perfrawdata_netframework_netclrexceptions"
```

Comment:
Leave out the `win32_perfrawdata_` here as well

Reply:
Perhaps even shorten to just `netframework_exceptions`?

Reply:
Yes good point! I changed it to "netframework_clrexceptions"